plugins/optimization-detective/helper.php

```diff
@@ -60,6 +60,40 @@ function od_generate_media_query( ?int $minimum_viewport_width, ?int $maximum_vi
 	}
 }
 
+/**
+ * Gets the reasons why Optimization Detective is disabled for the current request.
```

Comment:
Maybe a tiiiiny bit more accurate:

```suggestion
 * Gets the reasons why Optimization Detective is disabled for the current response.
```